ivy/functional/ivy/gradients.py

```diff
@@ -325,6 +325,7 @@ def execute_with_gradients(func, xs, retain_grads=False):
     """
     return current_backend(None).execute_with_gradients(func, xs, retain_grads)
 
+setattr(execute_with_gradients, "computes_gradients", True)
```

Comment:
`execute_with_gradients.computes_gradients = True` is cleaner, same in other locations across the code.